src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -0,0 +1,523 @@
+                            </div>
+                            <div
+                                className={`${classes.inputSectionWrapStyles} ${classes.wrapStyles}`}
+                            >
+                                <Typography
+                                    component="h2"
+                                    className={classes.titleStyles}
+                                >
+                                    Number of Workers
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Select a number or a range for the number of
+                                    people employed at the location. For
+                                    example: 100, 100-150.
+                                </Typography>
+                                <StyledSelect
+                                    id="number_of_workers"
+                                    name="Number of Workers"
+                                    options={numberOfWorkersOptions || []}
+                                    value={numberOfWorkers}
+                                    onChange={setNumberOfWorkers}
+                                    placeholder="Enter the number of workers as a number or range"
+                                    aria-label="Number of workers"
+                                    isMulti={false}
+                                    styles={selectStyles}
+                                    className={classes.selectStyles}
+                                />
+                            </div>
+                            <div
+                                className={`${classes.inputSectionWrapStyles} ${classes.wrapStyles}`}
+                            >
+                                <Typography
+                                    component="h2"
+                                    className={classes.titleStyles}
+                                >
+                                    Parent Company
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Enter the company that holds majority
+                                    ownership for this production.
+                                </Typography>
+                                <StyledSelect
+                                    creatable
+                                    name="Parent company"
+                                    value={parentCompany}
+                                    onChange={setParentCompany}
+                                    placeholder="Select"
```

Comment:
Please change the placeholder to a more meaningful one (e.i. Enter the parent company).

Reply:
done